lib/output.js

```diff
@@ -887,6 +904,21 @@ function OutputStream(options) {
         self._do_print(output, "throw");
     });
 
+    /* -----[ yield ]----- */
+
+    DEFPRINT(AST_Yield, function(self, output){
+        var star = self.is_star ? "*" : "";
+        output.print("yield" + star);
+        if (self.expression) {
+            output.space();
+            self.expression.print(output);
+        } else if (self.is_star === true) {
```

Comment:
meh

```js
var things = [ (... some content ...)];
for (var i = 0; i < 10; i++) {
    yield things[i];
}
```

vs

```js
var things = [ (... some content) ... ];
yield* things;
```

Provided the js interpreter knows that array is iterable. It's a handy syntax sugar.